sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -776,6 +777,10 @@ void gnrc_ipv6_netif_init_by_dev(void)
 
             DEBUG("ipv6 netif: Set 6LoWPAN flag\n");
             ipv6_ifs[i].flags |= GNRC_IPV6_NETIF_FLAGS_SIXLOWPAN;
+#ifdef MODULE_GNRC_SIXLOWPAN_ND_ROUTER
+            DEBUG("ipv6 netif: Set router flag\n");
+            ipv6_ifs[i].flags |= GNRC_IPV6_NETIF_FLAGS_ROUTER;
```

Comment:
`gnrc_ipv6_netif_set_router()` is called in L830 which does the same thing (+ other stuff) already.